frontend/src/utilities/segmentIOUtils.tsx

```diff
@@ -96,7 +96,9 @@ export const initSegment = async (props: {
       const t = document.createElement('script');
       t.type = 'text/javascript';
       t.async = true;
-      t.src = `https://cdn.segment.com/analytics.js/v1/${encodeURIComponent(key)}/analytics.min.js`;
+      t.src = `https://console.redhat.com/connections/cdn/analytics.js/v1/${encodeURIComponent(
```

Comment:
this will not work if access to console.redhat.com is not possible directly. As is the case in any corporate setup ... i.e. HTTP_PROXY and HTTPS_PROXY env vars need to be set and honored by your connections here. 

https://docs.openshift.com/container-platform/4.14/networking/enable-cluster-wide-proxy.html#nw-proxy-configure-object_config-cluster-wide-proxy

https://docs.openshift.com/container-platform/4.9/rest_api/config_apis/proxy-config-openshift-io-v1.html#apisconfig-openshift-iov1proxies



So that, from the proxy, if allowed, access continues to console.redhat.com.

Can this functionality be disabled? If not, this would in its current form be a breaking change. I am not talking public cloud and commercial Red Hat, I am talking ODH.